modules/monitoring/application/forms/Command/Object/ScheduleServiceDowntimeCommandForm.php

```diff
@@ -78,25 +120,28 @@ public function createElements(array $formData = array())
                     'required'      => true,
                     'label'         => $this->translate('Start Time'),
                     'description'   => $this->translate('Set the start date and time for the downtime.'),
-                    'value'         => $start
+                    'preserveDefault' => true,
+                    'value'         => $this->start
                 )
             ),
             array(
                 'dateTimePicker',
                 'end',
                 array(
-                    'required'      => true,
-                    'label'         => $this->translate('End Time'),
-                    'description'   => $this->translate('Set the end date and time for the downtime.'),
-                    'value'         => $end
+                    'required'        => true,
+                    'label'           => $this->translate('End Time'),
+                    'description'     => $this->translate('Set the end date and time for the downtime.'),
+                    'preserveDefault' => true,
+                    'value'           => $isFlexible ? $this->flexibleEnd : $this->fixedEnd
                 )
             ),
             array(
                 'select',
                 'type',
                 array(
                     'required'      => true,
-                    'autosubmit'    => true,
+                    'class'    => 'autosubmit',
+                    'preserveDefault' => true,
```

Comment:
Please keep using `autosubmit => true` in Icinga Web 2. It's only `ipl\Html\Form` which can't handle this (yet?).

And.. ensure that all `=>` are equally positioned in array definitions. :wink: